js/components/table-container.tsx

```diff
@@ -185,17 +185,19 @@ class Table extends React.Component<ITableContainerProps, ITableContainerState>
                 this.props.id, model.fileDataRequest,
                 this.props.flowKey,
                 manywho.settings.global('paging.table'),
-                state.search,
+                state ? state.search : '',
                 null,
                 null,
-                state.page,
+                state ? state.page : 1,
             );
         }
     }
 
     componentDidMount() {
         window.addEventListener('resize', this.handleResizeDebounced);
-        this.fetchFiles();
+        if (!this.props.isDesignTime) {
```

Comment:
On line 269, if we pass down a prop called isDesignTime with a value of this.props.isDesignTime, then I think that will disable the file uploader fully.
Currently the dropzone works, which feels weird.
It looks like in IFileUploadProps, not all of the props that the component uses are defined: e.g. isDesignTime, flowKey.

Reply:
ok, I'll sort that too.

Reply:
thanks